force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +92,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> splitSentence = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWord = new Set<String>();
 
         // Add each word to the set
+        uniqueWord.addAll(splitSentence);
 
         // Join the unique words back into a sentence
+        List<String> wordList = new  List<String>();
+        for (String word : uniqueWord){
+            wordList.add(word);
+        }
+
+        String newSentence = String.join(wordList,' ');
```

Comment:
Great use of String.join(), remember this method because it is insanely useful. 

You actually don't need to create a new List to rejoin the words; you can use String.join with a Set as long as you are using the latest API version.